commit/plugin.go

```diff
@@ -384,11 +430,40 @@ func (p *Plugin) Outcome(
 		MerkleRootOutcome: merkleRootOutcome,
 		TokenPriceOutcome: tokenPriceOutcome,
 		ChainFeeOutcome:   chainFeeOutcome,
+		MainOutcome:       p.getMainOutcome(outCtx, prevOutcome, tokenPriceOutcome, chainFeeOutcome),
 	}
 	p.metricsReporter.TrackOutcome(out)
 	return out.Encode()
 }
 
+func (p *Plugin) getMainOutcome(
+	outCtx ocr3types.OutcomeContext,
+	prevOutcome committypes.Outcome,
+	tokenPriceOutcome tokenprice.Outcome,
+	chainFeeOutcome chainfee.Outcome,
+) committypes.MainOutcome {
+	pricesObservedInThisRound := len(tokenPriceOutcome.TokenPrices) > 0 || len(chainFeeOutcome.GasPrices) > 0
+	if pricesObservedInThisRound {
+		return committypes.MainOutcome{
+			InflightPriceOcrSequenceNumber: cciptypes.SeqNum(outCtx.SeqNr),
+			RemainingPriceChecks:           p.offchainCfg.InflightPriceCheckRetries,
```

Comment:
nit: reorder to have `RemainingPriceChecks` first to match below two MainOutcome instatiations